war/src/main/webapp/css/style.css

```diff
@@ -1,18 +1,18 @@
 /*
  * The MIT License
- * 
+ *
```

Comment:
ok I removed the spaces. but someone should clean that up in a separate pr maybe...